src/Evolution/SetPDFSet.f

```diff
@@ -15,46 +15,48 @@ subroutine SetPDFSet(name)
 *     Variables
 *
       integer ln
-c      character name*(*)
-      character*100 name
+      character name*(*)
```

Comment:
Unfortunately, this would not work.
I tried it already in the past and tried it again right now and does not work (at least on my machine).
If you try to feed an LHAPDF set it just does not see anything.

Reply:
I'm fine with the previous declaration, however we cannot perform the substring `if` as we do now because the original string is not size 100, so if you call `name(1:12)` it will deliver values out of memory. 

So my question is: fortran has some way to measure the real size of `name` e.g. `trim`, `len` or `strlen`?